docs/docker/installation/installation-using-docker-windows-10-lower.md

```diff
@@ -23,7 +23,7 @@ cd project-base
 ```
 Notes: 
 - The `--no-install` option disables installation of the vendors - this will be done later in the Docker container
-- If you want to keep the GIT history of `shopsys/project-base` in your new project, use the `--keep-vcs` option
+- Option `--keep-vcs` creates repository in your new project, that is needed for diff commands of application build and keeps the GIT history of `shopsys/project-base` in your new project
```

Comment:
The formulation "creates repository in your project" seems little bit confusing to me. I would put it this way: "Option --keep-vsc initializes GIT repository in the folder with your new project, that is needed...and keeps the GIT history of shopsys/project-base."